scripts/remover.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-FILES=$(ls *.js)
+FILES=$(ls ./*.js)
 
 for FILE in $FILES
 do
-	if [ -f $FILE ]
+	if [ -f "$FILE" ]
 	then
-		sed -i -e "/if (typeof Int64 === 'undefined' && typeof require === 'function') {/,+2 d" $FILE
+		sed -i -e "/if (typeof Int64 === 'undefined' && typeof require === 'function') {/,+2 d" "$FILE"
```

Comment:
This is a good change for safety if there might be a filename with a space in it, but then it's not related to this patch.